src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,26 @@
+.watch {
+  @include square ($watch-size);
+  border: 1px solid black;
+  background-color: #ccc;
+
+  &__second-line {
+    @include line ($second-line-width, $second-line-height);
+    background-color: orange;
```

Comment:
Use a variable for the background color 'orange' for better maintainability and consistency.